tests/e2e/e2e_test.go

```diff
@@ -119,6 +137,11 @@ var _ = AfterSuite(func() {
 
 var _ = Describe("odoe2e", func() {
```

Comment:
We should probably have these tests running separately under a different `Describe` spec?